frontend/src/pages/user/seller/allReports/index.tsx

```diff
@@ -0,0 +1,37 @@
+import { useState } from 'react';
+
+import TButton from '@components/TButton';
+import FormItem from '@components/FormItem';
+
+const SellerReport = () => {
+  const now = new Date();
+  const [year, setYear] = useState(now.getFullYear());
+  const [month, setMonth] = useState(now.getMonth());
+  return (
+    <div style={{ padding: '5% 8% 10% 8% ' }}>
+      <h3>At what time would you like to view the report?</h3>
+      <hr className='hr' />
+      <FormItem label='Year'>
+        <input
+          type='text'
+          defaultValue={year}
+          onChange={(e) => {
+            setYear(parseInt(e.target.value));
```

Comment:
Can u make it only able to input number just like the number type input?
I think allow user to enter anything then parse it into NaN then go to 404 is bad UX.
It shouldnt allow to do it at first or at least warn it before going to 404.